### ISSUE TITLE
Display both directions for quick conversions

### ISSUE THREAD

**Issue by hlaueriksson** [Advaith3600/PowerToys-Run-Currency-Converter#3]
It would be convenient to get both directions when using the quick conversions.

Example:
![image](https://github.com/Advaith3600/PowerToys-Run-Currency-Converter/assets/1299786/3f1ac1b5-d194-4236-958b-5e033cf2b5e1)

The `Quick Conversion Direction` setting could be used to select which direction to display first.

**Comment by Advaith3600** [comment]
That is an interesting idea, I will work on it. This will remove the need for the quick conversion direction. I will remove that as well.

**Comment by Advaith3600** [comment]
What is your opinion about the idea where if the user enters `1 eur`, it will list

``` 
1 eur to local
1 eur to global
1 local to eur
1 global to eur
```

or just

```
1 eur to local
1 eur to global
```

**Comment by Advaith3600** [comment]
I have released a new version [v1.0.3](https://github.com/Advaith3600/PowerToys-Run-Currency-Converter/releases/tag/v1.0.3). There are some more interesting additions.
